### ISSUE TITLE
Beacon not syncing

### ISSUE THREAD

**Issue by GeoJesse** [sigp/lighthouse#1590]
## Description

See issue #1560 
Upgraded to https://github.com/sigp/lighthouse/releases/tag/v0.2.8.  Purged the DB and now it won't sync at all.

## Version

0.2.8

## Present Behaviour

Beacon not syncing.  After a while the est_time goes to --.
Example log is:
Sep 05 14:05:50 ubuntu lighthouse[2351]: Sep 05 14:05:50.000 INFO Syncing                                 est_time: --, distance: 127464 slots (2 weeks 3 days), peers: 16, service: slot_notifier

## Expected Behaviour

It should sync.

## Steps to resolve

Unknown.  With 0.2.7 I was able to purge the DB and it would sync eventually with est_time decreasing.  Now it never catches up.

**Comment by tobisako** [comment]
ref) I'm tried this issue with lighthouse-docker
TESTNET=medalla -> Non-reproducible
TESTNET=altona -> reproduced

medalla
```
beacon_node_1       | Sep 09 06:48:21.811 INFO Syncing                                 est_time: 0 mins, speed: 17.31 slots/sec, distance: 62 slots (12 mins), peers: 43, service: slot_notifier
```

**Comment by GeoJesse** [comment]
I reverted back to 0.2.7 and it still won't sync.  I've purged the db several times but no luck.

Latest logs read as:
Sep 09 12:01:50 ubuntu lighthouse[5932]: Sep 09 12:01:50.001 INFO Syncing                                 est_time: --, distance: 151740 slots (3 weeks 0 days), peers: 25, service: slot_notifier

**Comment by james-c** [comment]
I'm getting the same problem with the docker image (sigp/lighthouse:latest). Get's to about 3 weeks to 3 weeks 5 days out and the chain stops syncing.

**Comment by AgeManning** [comment]
This is a known problem. There is a race condition syncing the period of long non-finality on medalla. Faster computers should be able to sync easily, slower computers have a probablistic chance. 

Even though the logs report no syncing time. Your node is still attempting to sync in the background. If you let it run, you should eventually sync (although it may take a while). 

We have a fix coming very soon.

**Comment by GeoJesse** [comment]
Glad to hear.  You all rock!

**Comment by AgeManning** [comment]
Hey guys. While we wait for the correct sync PR to land. I've made a hack-fix which should help most peers sync. 

You can use the `sync-hack` branch to sync to head :)

**Comment by AgeManning** [comment]
This is resolved in #1615. The correct full fix comes in #1628

**Comment by GeoJesse** [comment]
Sync'd in record time and hitting 100% attestation effectiveness!  Thanks!
